app/templates/application.hbs

```diff
@@ -81,45 +88,8 @@
       </li>
     {{/if}}
   </AuMainHeader>
-  {{#unless this.isIndex}}
-    <Shared::NavigationBar />
-  {{/unless}}
-  {{#if this.session.isAuthenticated}}
-    <AuMainContainer as |main|>
```

Comment:
doesn't this mean the maincontainer is gone in all other routes than the overzicht route? please keep it around on the application level. It's there so appuniversum can put general app stuff in this component

Reply:
Added it again